src/implementations/twig/components/site-header/ec-site-header.story.js

```diff
@@ -73,85 +74,155 @@ const getArgTypes = (data) => {
   if (data.login_box) {
     argTypes.show_login = {
       name: 'login',
-      type: { name: 'boolean' },
+      control: { type: 'boolean' },
       description: 'Show the login box',
       table: {
+        type: { summary: 'boolean' },
```

Comment:
same concept, toggles that we define for storybook only should not have a table set, but in this case we could use "login box" as the label and in that case we could define the table based on the corresponding parameter, which is an empty object by default.